### ISSUE TITLE
Update logo files to higher resolution SVG file and make consistent between website and docs

### ISSUE THREAD

**Issue by rgommers** [scipy/scipy#14241]
Our logo is really old, and the history for how it got created got lost. It's even unclear what the "official" logo is - we have low-resolution PNGs in two colors that we use today:
- Red on [scipy.org](https://scipy.org/): https://github.com/scipy/scipy.org/blob/master/www/_static/images/scipy_med.png
- Blue in the [docs](https://scipy.github.io/devdocs/): https://github.com/scipy/scipy/blob/master/doc/source/_static/scipyshiny_small.png

@stefanv somehow managed to dig up a higher-res blue version including an SVG file at:
- https://github.com/scikit-image/skimage-branding/blob/main/logo/data/scipy.png
- https://github.com/scikit-image/skimage-branding/blob/main/logo/scipy.svg

There's a long-ish mailing list thread here about it (warning: archive is weird, thread is split over April/May):
- https://mail.python.org/pipermail/scipy-dev/2019-April/023482.html
- https://mail.python.org/pipermail/scipy-dev/2019-May/023490.html

All the above is context because this is hard to find in one place. Now the actual task: take the high-res SVG file, and use it in both the docs and on the website. The links above should already show which files to replace.

**Comment by rgommers** [comment]
Argh, except the SVG file is not the same, it looks more boring. I think we'd like an SVG version of the blue logo that we now use in the docs. Changing the task to recreating that.

**Comment by stefanv** [comment]
@rgommers Is this the image that you'd like to recreate?

https://docs.scipy.org/doc/scipy/reference/_static/img/scipy_org_logo.png

I see that logo differs modestly from the one above (in shape).

I will also reach out to Enthough to find out if they have copies of any of the vector drawings used.

**Comment by charris** [comment]
I believe the originals were by Travis Vaught.

**Comment by rgommers** [comment]
No, this one: https://github.com/scipy/scipy/blob/master/doc/source/_static/scipyshiny_small.png

![image](https://user-images.githubusercontent.com/98330/122245283-e3c2b680-cec5-11eb-9dc1-6a25c9d201b7.png)

I _think_ that's our "official" logo.

**Comment by charris** [comment]
@rgommers Hey, where did the python's tongue go? That png is also available in old numpy versions.

**Comment by stefanv** [comment]
The shiny button-style logos were the originals; the SciPy conference then renewed them to be "flat" (and I think those are the more modern looking ones we should use). You'll see similar button-style logos on the SciPy frontpage:

![2021-06-16_08:39:25](https://user-images.githubusercontent.com/45071/122250275-6031a700-ce7e-11eb-961d-8d28e2d165fd.png)

**Comment by rgommers** [comment]
So you like https://raw.githubusercontent.com/scikit-image/skimage-branding/main/logo/scipy.svg @stefanv?

<img width="405" alt="image" src="https://user-images.githubusercontent.com/98330/122294160-108fc180-cef8-11eb-9698-c69eb9b6a00f.png">

It seems a little "sparse" to me. Maybe the flat think makes sense, but not even having a darker line around the circle makes it look like it misses something.

**Comment by stefanv** [comment]
That's the one, yes.  We could modify it, of course!

**Comment by ilayn** [comment]
There are some problems with the logo glat or shiny, namely

- The head and tail angles are off, 
- it's probably made out of an S shape inverted hence top and bottom parts are different sizes and tilted to the left
- just middle part of is modified but not finished so there are a couple of kinks once seen difficult to unsee 
- python head is also not symmetric and looks random
- the tongue comes out not from the center of the mouth

and some other minor issues 

![image](https://user-images.githubusercontent.com/1303842/122640393-8fa31680-d0ff-11eb-9c6c-d97ca4bd2875.png)

![image](https://user-images.githubusercontent.com/1303842/122640897-2a045980-d102-11eb-8148-2d848ebca5f8.png)

We can just fix some of these quickly but better would be letting a professional go at it.

**Comment by tupui** [comment]
Great to see this issue!

Would it be an option to contact the person who is doing the logos for SciPy conf?? They have nice designs all the time and maybe we could take the opportunity to modernize it?

**Comment by stefanv** [comment]
That person is Erick Michaud.

He also provided me with the other SciPy Conference logos, for reference.  I'll attach all the logos I've received here.

[scipy-logos.zip](https://github.com/scipy/scipy/files/6681438/scipy-logos.zip)

**Comment by tupui** [comment]
Thanks @stefanv! Do we have the authorization to use/modify these as we please?

Also agree with @ilayn that having a professional look at this would be a good thing. We could ask to clarify the whole branding (and could include guidelines for the website). We could use a SDG for instance (and currently there is a general lack of proposals...).

**Comment by stefanv** [comment]
I think we are fine to use and modify the first logo (the one also posted by Ralf above), but not the conference versions.

**Comment by stefanv** [comment]
I've received confirmation from Enthough that they have copyright on all the logos, but that we are free to modify and use the SciPy logo.

**Comment by tupui** [comment]
> I've received confirmation from Enthough that they have copyright on all the logos, but that we are free to modify and use the SciPy logo.

To be clear, SciPy logo here means all the zip you posted or just the one mentioned by Ralf? (both shiny and not shiny?)

**Comment by stefanv** [comment]
All these logos are copyrighted by Enthought, AFAIK.

**Comment by lucascolley** [comment]
From scipy/archive#2, it looks like this issue should either be renamed to "find a designer [for a new logo]", or closed.
